spotibetical.rb

```diff
@@ -17,8 +18,8 @@
 class Spotibetical < Sinatra::Base
 
   enable :sessions, :method_override
-  set :session_secret, 'super secret'
-
+  set :session_secret, Digest::SHA1.hexdigest(Time.now.to_s)
```

Comment:
This would be secure but has a problem: everytime we restart the server (like when we deploy a new version), we'll get a new session secret, and so all the existing sessions will be invalid, and all users will be logged out. I think this needs to be an envrionment variable.